### ISSUE TITLE
[ISSUE] new_cluster field is empty when creating databricks_job

### ISSUE THREAD

**Issue by Gnarik** [databrickslabs/databricks-terraform#75]
Hi there,

When we are trying to create a Databricks job with the new_cluster field fulfilled the payload sent to the API is empty.

### Affected Resource(s)
- databricks_job

### Terraform Configuration Files
```hcl
provider "databricks" {
  host = "https://xxxxxx.cloud.databricks.com/"
  token = "xxxxxx"
}

resource "databricks_job" "my_job3" {
  new_cluster  {
    autoscale  {
      min_workers = 2
      max_workers = 3
    }
    spark_version = "6.4.x-scala2.11"
    aws_attributes  {
      availability = "SPOT"
      zone_id = "us-east-1a"
      spot_bid_price_percent = "100"
    }
    node_type_id = "r3.xlarge"
  }
  notebook_path = "/Users/jane.doe@databricks.com/my-demo-notebook"
  name = "my-demo-notebook"
  timeout_seconds = 3600
  max_retries = 1
  max_concurrent_runs = 1
}
```

### Debug Output
https://gist.github.com/Gnarik/21805a0ceb7e8fd26b67318d83ff80f6

### Expected Behavior
We would expect to have the field new_cluster fulfilled with the values specified in the terraform script then a job that spin up a new cluster when it starts is created on the Databricks environment.

### Actual Behavior
The new_cluster field is empty in the API HTTP payload and no job is created

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform init`
1. `terraform apply`

### Important Factoids
None

### References
None

**Comment by stikkireddy** [comment]
@Gnarik hey is this with the released build or the master branch build?

**Comment by Gnarik** [comment]
Hi, this is the HEAD of the master branch. Note sure it's relevant but note that I only ran the following command from the Makefile:
`@go build -mod vendor -v -o terraform-provider-databricks`

**Comment by Gnarik** [comment]
Fixed by the #79

**Comment by Gnarik** [comment]
Thank you very much for the very fast correction.
